README.md

```diff
@@ -116,6 +116,136 @@ notifications):
 script: bundle exec rake run_about_yml_check
 ```
 
+
+## `.about.yml` cheat sheet
+The following attributes are currently stored in the `.about.yml` file and are used in one or more of the locations referenced above. Required attributes are marked with an asterix, and field descriptions and examples are offered. Take a look at [Every Kid in a Park](https://github.com/18F/ekip-api/blob/master/.about.yml) or [Open Opportunities](https://github.com/18F/openopps-platform/blob/dev/.about.yml) for living examples of `.about.yml` in action.
+
+
+`name`* - This is a short name of your project that is used as a URL slug on the 18F dashboard.
+> 
+```yml
+name: ekip-api
+```
+
+`full_name`* - This is the display name of your project on the 18F dashboard.
+> 
+```yml
+full_name: Every Kid in a Park
+```
+
+`description`* - What is the problem your project solves? What is the solution? The #dashboard team will gladly help you put this together for your project.
+> 
+```yml
+description: In 2015, President Obama formally announced the Every Kid in a Park program, which
```

Comment:
The formatting for this and other long-form text fields should be broken across multiple lines and indented like this:

```yaml
description: |
  In 2015, President Obama formally announced the Every Kid in a Park program,
  which...
```